### ISSUE TITLE
output spatial resolution selection

### ISSUE THREAD

**Issue by mdelgadoblasco** [ASFHyP3/ASFHyP3.github.io#9]
Could the user select the output spatial resolution thinner than 40m/80m ?
Maybe selecting the multilook factor 5x1 instead of 10x2?

Thank you very much

**Comment by jhkennedy** [comment]
Hi @mdelgadoblasco you can select 5x1 look using the `INSAR_ISCE_BURST` job type: 
<https://hyp3-docs.asf.alaska.edu/guides/burst_insar_product_guide/>

**Comment by jhkennedy** [comment]
@mdelgadoblasco correct, we do not offer 5x1 for INSAR_GAMMA. Now that we've moved to a credit system, we could offer it, but I don't know of a timeline for doing so. CC @krkristenson-asf 

for 5x1, the output pixel spacing will be 20m, which closely matches the resolution of the multilooked images. Providing an option for different pixel spacings would not give you significantly better results than resampling the HyP3 products to your desired pixel spacing.
